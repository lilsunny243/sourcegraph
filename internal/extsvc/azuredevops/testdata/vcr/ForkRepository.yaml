---
version: 1
interactions:
- request:
    body: '{"name":"sgtestazureforks","project":{"id":"dc493f7d-0b57-4de2-a59b-3f74ff3ea334"},"parentRepository":{"id":"c4d186ef-18a6-4de4-a610-aa9ebd4e1faa","project":{"id":"dc493f7d-0b57-4de2-a59b-3f74ff3ea334"}}}'
    form: {}
    headers:
      Content-Type:
      - application/json
    url: https://dev.azure.com/sgtestazure/_apis/git/repositories?api-version=7.0
    method: POST
  response:
    body: '{"id":"edaf746e-a05e-429c-ad75-2486f9c0f7d7","name":"sgtestazureforks","url":"https://dev.azure.com/sgtestazure/dc493f7d-0b57-4de2-a59b-3f74ff3ea334/_apis/git/repositories/edaf746e-a05e-429c-ad75-2486f9c0f7d7","project":{"id":"dc493f7d-0b57-4de2-a59b-3f74ff3ea334","name":"sgtestazure","url":"https://dev.azure.com/sgtestazure/_apis/projects/dc493f7d-0b57-4de2-a59b-3f74ff3ea334","state":"wellFormed","revision":11,"visibility":"private","lastUpdateTime":"2023-01-20T19:22:47.403Z"},"size":0,"remoteUrl":"https://sgtestazure@dev.azure.com/sgtestazure/sgtestazure/_git/sgtestazureforks","sshUrl":"git@ssh.dev.azure.com:v3/sgtestazure/sgtestazure/sgtestazureforks","webUrl":"https://dev.azure.com/sgtestazure/sgtestazure/_git/sgtestazureforks","isFork":true,"_links":{"self":{"href":"https://dev.azure.com/sgtestazure/dc493f7d-0b57-4de2-a59b-3f74ff3ea334/_apis/git/repositories/edaf746e-a05e-429c-ad75-2486f9c0f7d7"},"project":{"href":"vstfs:///Classification/TeamProject/dc493f7d-0b57-4de2-a59b-3f74ff3ea334"},"web":{"href":"https://dev.azure.com/sgtestazure/sgtestazure/_git/sgtestazureforks"},"ssh":{"href":"git@ssh.dev.azure.com:v3/sgtestazure/sgtestazure/sgtestazureforks"},"commits":{"href":"https://dev.azure.com/sgtestazure/dc493f7d-0b57-4de2-a59b-3f74ff3ea334/_apis/git/repositories/edaf746e-a05e-429c-ad75-2486f9c0f7d7/commits"},"refs":{"href":"https://dev.azure.com/sgtestazure/dc493f7d-0b57-4de2-a59b-3f74ff3ea334/_apis/git/repositories/edaf746e-a05e-429c-ad75-2486f9c0f7d7/refs"},"pullRequests":{"href":"https://dev.azure.com/sgtestazure/dc493f7d-0b57-4de2-a59b-3f74ff3ea334/_apis/git/repositories/edaf746e-a05e-429c-ad75-2486f9c0f7d7/pullRequests"},"items":{"href":"https://dev.azure.com/sgtestazure/dc493f7d-0b57-4de2-a59b-3f74ff3ea334/_apis/git/repositories/edaf746e-a05e-429c-ad75-2486f9c0f7d7/items"},"pushes":{"href":"https://dev.azure.com/sgtestazure/dc493f7d-0b57-4de2-a59b-3f74ff3ea334/_apis/git/repositories/edaf746e-a05e-429c-ad75-2486f9c0f7d7/pushes"},"forkSyncOperation":{"href":"https://dev.azure.com/sgtestazure/_apis/git/repositories/edaf746e-a05e-429c-ad75-2486f9c0f7d7/forkSyncRequests/9"}},"isDisabled":false,"isInMaintenance":false}'
    headers:
      Access-Control-Expose-Headers:
      - Request-Context
      Activityid:
      - 7b6e99fd-2d22-4f8e-9c2e-a8969845bdf3
      Cache-Control:
      - no-cache, no-store, must-revalidate
      Content-Length:
      - "2163"
      Content-Type:
      - application/json; charset=utf-8; api-version=7.0
      Date:
      - Fri, 17 Feb 2023 01:31:49 GMT
      Expires:
      - "-1"
      P3p:
      - CP="CAO DSP COR ADMa DEV CONo TELo CUR PSA PSD TAI IVDo OUR SAMi BUS DEM NAV
        STA UNI COM INT PHY ONL FIN PUR LOC CNT"
      Pragma:
      - no-cache
      Request-Context:
      - appId=cid-v1:72d31d95-1757-44f5-b910-a46611808454
      Strict-Transport-Security:
      - max-age=31536000; includeSubDomains
      X-Cache:
      - CONFIG_NOCACHE
      X-Content-Type-Options:
      - nosniff
      X-Frame-Options:
      - SAMEORIGIN
      X-Msedge-Ref:
      - 'Ref A: F61ED5507AD441A296292A681ECA6E97 Ref B: YTO01EDGE0719 Ref C: 2023-02-17T01:31:49Z'
      X-Tfs-Processid:
      - c208d684-a461-41e5-a43d-ea1d99a0087b
      X-Tfs-Session:
      - 7b6e99fd-2d22-4f8e-9c2e-a8969845bdf3
      X-Vss-E2eid:
      - 7b6e99fd-2d22-4f8e-9c2e-a8969845bdf3
      X-Vss-Senderdeploymentid:
      - 4ff21e82-8865-0b2e-ffe8-9598818f8190
      X-Vss-Userdata:
      - 473dec3e-03d7-6147-b106-0b0a7f766a92:idan.varsano@sourcegraph.com
    status: 201 Created
    code: 201
    duration: ""
