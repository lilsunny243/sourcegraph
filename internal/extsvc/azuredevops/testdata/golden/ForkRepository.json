{
  "id": "edaf746e-a05e-429c-ad75-2486f9c0f7d7",
  "name": "sgtestazureforks",
  "remoteURL": "https://sgtestazure@dev.azure.com/sgtestazure/sgtestazure/_git/sgtestazureforks",
  "url": "https://dev.azure.com/sgtestazure/dc493f7d-0b57-4de2-a59b-3f74ff3ea334/_apis/git/repositories/edaf746e-a05e-429c-ad75-2486f9c0f7d7",
  "sshUrl": "git@ssh.dev.azure.com:v3/sgtestazure/sgtestazure/sgtestazureforks",
  "webUrl": "https://dev.azure.com/sgtestazure/sgtestazure/_git/sgtestazureforks",
  "isDisabled": false,
  "isFork": true,
  "project": {
   "id": "dc493f7d-0b57-4de2-a59b-3f74ff3ea334",
   "name": "sgtestazure",
   "state": "wellFormed",
   "revision": 11,
   "visibility": "private",
   "url": "https://dev.azure.com/sgtestazure/_apis/projects/dc493f7d-0b57-4de2-a59b-3f74ff3ea334"
  }
 }